test/specs/validate-address.spec.js

```diff
@@ -267,7 +267,7 @@ describe("validateAddress()", () => {
     const shipengine = new ShipEngine({ apiKey, baseURL });
 
     const addressToValidate = {
-      street: ["上鳥羽角田町６８", "validate-with-non-latin-chars"],
+      street: ["上鳥羽角田町６８"],
```

Comment:
I'm not sure how this was passing in the old Simengine implementation. In the current version, this correlates to a fixture but only without the "validate-with-non-latin-chars" part of the address.

Reply:
The "validate-with-non-latin-chars" should have been removed earlier. It's a leftover from the first version of SimEngine which used that to match to a fixture.